iam/src/procedures/githubOauth.ts

```diff
@@ -0,0 +1,54 @@
+import crypto from "crypto";
+import axios from "axios";
+
+export const getSessionKey = (): string => {
```

Comment:
is this session key needed? i don't see a need to maintain state over a verification "session", unless you add in a check to ensure the "session key" generated here, matches the `state` passed back from the github redirect.

otherwise it seems like everything here could be moved into the `providers/github.ts` file, since it's only used during the `verifyGithub` process.

Reply:
The session key was not needed (that was used as the state parameter). The creation & verification of state parameter has been move to front-end.
I have removed the filed. The functions that are still required were moved to  `providers/github.ts`.